tests/conftest.py

```diff
@@ -79,18 +79,26 @@ def add_slurm_user(user, container):
     except:
         raise
 
+def link_python(container):
+    cmd = "ln -s $(which python3.6) /usr/bin/python3"
```

Comment:
Does this work? On my system a double-$$ is needed: 

cmd = "ln -s $$(wich python3.6) /usr/bin/python3"

Reply:
I'll add this change in a coming PR. Merging now.